sbpy/data/names.py

```diff
@@ -475,14 +476,16 @@ def parse_asteroid(s):
                # number [7,8]
                '|^(([1-9][0-9]*A))'
                # comet-style designations: 1A [10]
+               '|^A/([12][0-9][0-9][0-9] [A-Z][0-9]+)'
+               # asteroids with cometary orbits [12]
                )
 
         # regex patterns that will be rejected
-        rej_pat = ('([1-2][0-9]{0,3}[ _][A-Z][0-9]*(\b|$))'
+        rej_pat = ('([CPXD]/[1-2][0-9]{0,3}[ _][A-Z][0-9]*(\b|$))'
                    # comet desig
-                   '|([1-9][0-9]*[PDCXAI]\b)'
+                   '|([1-9][0-9]*[PDCXI]\b)'
```

Comment:
Again, same comment about "I" type.

Reply:
And here, since these patters are rejecting things that are not asteroidal, I think we want the I.  I've fixed line 484 and updated the comments.